stable/artifactory/values.yaml

```diff
@@ -7,6 +7,10 @@
 # Common
 initContainerImage: "alpine:3.8"
 
+installer:
```

Comment:
I think it would be nicer to have the defaults in the values.yaml file. @jainish
But that’s just my opinion

Reply:
@danielezer We want to get values even if someone comments it from `values.yaml`.

Reply:
@jainishshah17 ok, I understand your point. I just think it’s easier to see all the default values in one place. Anyway, I approved the PR, LGTM